_episodes_rmd/01-starting-with-data.Rmd

```diff
@@ -240,6 +240,27 @@ dim(dat)
 
 This tells us that our data frame, `dat`, has `r nrow(dat)` rows and `r ncol(dat)` columns.
 
+At the moment, the columns of our data frame are named as they were in the csv file `inflammation-01.csv`. We can find the names of `dat` using the command `names`:
+
+```{r}
+names(dat)
+```
+
+We know that each row represents one patient and each column represents one time point. It is a good programing practice to use column names that indicate as good as possible the nature of the data. In our case, columns represent time points. Thus, let's modify our column names to T1, T2, T3 and so on. For this, we will use a comman called `gsub`:
```

Comment:
As @chendaniely recommended (I agree), please remove this section and below about changing the column names.

However, your suggestion for a better column naming could be implemented in the raw data files, and highlighted here.